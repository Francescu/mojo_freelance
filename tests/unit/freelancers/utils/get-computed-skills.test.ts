import moment from "moment";
import getComputedSkills from "../../../../src/freelancers/utils/get-computed-skills";

const skillsExperiences = {
  Java: {
    dates: [
      {
        endDate: moment.parseZone("2016-09-01T00:00:00+01:00"),
        startDate: moment.parseZone("2014-01-01T00:00:00+01:00"),
      },
      {
        endDate: moment.parseZone("2014-07-01T00:00:00+01:00"),
        startDate: moment.parseZone("2013-05-01T00:00:00+01:00"),
      },
    ],
    id: 400,
  },
};

describe("getComputedSkills", () => {
  test("It should return computed skills", () => {
    const result = getComputedSkills(skillsExperiences);

    expect(result).toMatchSnapshot();
  });
});
