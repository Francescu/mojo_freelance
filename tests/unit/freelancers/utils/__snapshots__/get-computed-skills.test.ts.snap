// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getComputedSkills It should return computed skills 1`] = `
Array [
  Object {
    "durationInMonths": 40,
    "id": 400,
    "name": "Java",
  },
]
`;
